model/metrics/nodejs-metrics.yaml

```diff
@@ -0,0 +1,98 @@
+groups:
+  - id: metric.nodejs.eventloop.delay.min
+    type: metric
+    metric_name: nodejs.eventloop.delay.min
+    brief: "Event loop minimum delay."
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    prefix: nodejs.eventloop.delay.min
+    attributes:
+      - ref: nodejs.version
```

Comment:
I think `nodejs.version` would be better be a resource attribute. Like https://github.com/open-telemetry/semantic-conventions/blob/main/model/resource/webengine.yaml#L9

Reply:
I had this as a resource attribute, and then a prior comment on this PR suggested as its own metric: 
> Also i think for version we need to set metric like it does in the prom-client library. Attribute in all metrics with version, for my opinion, it's not a good solution.

I'm open to suggestions on which option is best, I have nos strong preferences either way

Reply:
Are you referring to https://github.com/open-telemetry/semantic-conventions/pull/991#issuecomment-2093541602? I think @pikalovArtemN is suggesting that making a constant metric for resource attributes rathar than including these attributes in every metrics, maybe they can elaborate on their idea.

Reply:
I just realized that I read your initial comment wrong, my mistake. I agree it makes more sense. I made the changes to be a resource attribute now, let me know what you think 😄

Reply:
I'd think `process.runtime.version` is sufficient to represent nodejs version.

Reply:
good point! Removed